plopfile.mjs

```diff
@@ -0,0 +1,56 @@
+import path from "node:path";
+import * as glob from "glob";
+import autocompletePrompt from "inquirer-autocomplete-prompt";
+
+const PACKAGES_NEXTJS = "./packages/nextjs";
+
+/**
+ * @param {import("plop").NodePlopAPI} plop
+ */
+export default function(plop) {
+  plop.setPrompt("autocomplete", autocompletePrompt);
+  plop.setGenerator("stories", {
+    description: "Generates a Storybook stories file for a component",
+    prompts: [
+      {
+        name: "COMPONENT",
+        message: "Create a Stories file for which component? (you can search with wildcards)",
+        type: "autocomplete",
+        loop: false,
+        async source(answers, searchTerms) {
+          return glob.sync(`components/**/${searchTerms || ""}*`, {
+            cwd: PACKAGES_NEXTJS,
+            nodir: true,
+            nocase: true
+          }).map(file => {
+            const COMPONENT = {
+              FILE: file,
+              DIRNAME: path.dirname(file),
+              BASENAME: path.basename(file, path.extname(file)),
+            };
+            return {
+              name: file,
+              value: COMPONENT,
+            };
+          });
```

Comment:
🔧 It might be worth sorting this list for usability:
```suggestion
          }).sort((a, b) => a.name.localeCompare(b.name));
```

Reply:
Glob sorts by default 👍

Reply:
I guess it's a reverse sort then!
![image](https://github.com/FormidableLabs/nextjs-sanity-fe/assets/25199713/a94e0a31-f731-49a3-a13c-202981be92b6)

Reply:
Oh wow, can't believe I didn't notice that! Sorry, should've added your changes! I'll update it.